README.kafka

```diff
@@ -0,0 +1,127 @@
+
+   MAGPIE_LOCAL_DIR : For scratch space files
+
+   MAGPIE_JOB_TYPE : This should be set to 'kafka'
+
+   JAVA_HOME : B/c you need to ...
+
+3) Setup the essentials for Kafka.
+
+   KAFKA_SETUP : Set to yes
+
+   KAFKA_VERSION : Set appropriately.
+
+   KAFKA_HOME : Where your Kafka code is.  Typically in an NFS mount.
+
+   KAFKA_LOCAL_DIR : A small place for conf files and log files local to
+   each node.  Typically /tmp directory.
+
+4) Select how your job will run by setting KAFKA_MODE.  The first
+   time you'll probably want to run w/ 'performance' mode just to try
+   things out and make things look setup correctly.
+
+   After this, you may want to run with 'server' mode to play
+   around and figure things out.  In the job output you will see
+   output similar to the following:
+
+      ssh node70
+      setenv KAFKA_CONF_DIR "/tmp/achu/kafka/ajobname/1081559/conf"
+
+   These instructions will inform you how to login to the master node
+   of your allocation and how to initialize your session.  Once in
+   your session.  You can do as you please.  For example, you can
+   launch a kafka consumer (bin/kafka-console-consumer.sh ...).  There 
+   will also be instructions in your job output on how to tear the 
+   session down cleanly if you wish to end your job early.
+
+   See "Exported Environment Variables" in README for information on
+   common exported environment variables that may be useful in
+   scripts.
+
+   See below in "Kafka Exported Environment Variables", for
+   information on Kafka specific exported environment variables that
+   may be useful in scripts.
+
+5) Kafka requires Zookeeper, so ensure Zookeeper is configured and also in
+   your submission script.  See README.zookeeper for setup instructions.
+
+6) Submit your job into the cluster by running "sbatch -k
+   ./magpie.sbatchfile" for Slurm, "msub ./magpie.msubfile" for
+   Moab, or "bsub < .magpie.lsffile" for LSF.
+   Add any other options you see fit.
+
+7) Look at your job output file to see your output.  There will also
+   be some notes/instructions/tips in the output file for viewing the
+   status of your job in a web browser, environment variables you wish
+   to set if interacting with it, etc.
+
+   See "General Advanced Usage" in README for additional tips.
+
+Kafka Exported Environment Variables
+---------------------------------------
+
+The following environment variables are exported when your job is run
+and may be useful in scripts in your run or in pre/post run scripts.
+
+KAFKA_CONF_DIR : the directory that Kafka configuration files local
+                   to the node are stored
+
+KAFKA_LOG_DIR : the directory Kafka log files are stored
+
+KAFKA_LOG4J_OPTS : options variable for logging
+
+LOG_DIR : the directory to store GC logging
+
+See "Zookeeper Exported Environment Variables" in README.zookeeper,
+for Zookeeper environment variables that may be useful.
+
+Kafka Patching
+----------------
+No extra patching needed at this time.
+
```

Comment:
isn't there a no-local-dir patch below?  You can cut & paste some of the dialogue from the README.hadoop file that has language on no-local-dir.